tests/test_analysis.py

```diff
@@ -64,6 +64,25 @@ def test_can_find_python_files_in_dot(self):
             actual_suffix = os.path.splitext(python_path)[1]
             assert actual_suffix == ".py"
 
+    def test_can_find_files_from_cloned_git_link(self):
+        git_link = "https://github.com/roskakori/pygount.git"
+        with analysis.SourceScanner([git_link]) as scanner:
+            actual_paths = list(scanner.source_paths())
+            assert actual_paths != []
+
+    def test_can_split_repo_link_and_branch(self):
+        git_link_https_1 = "https://github.com/roskakori/pygount.git"
```

Comment:
I don't think these variables add anything to making the code easier to understand. Might as well pass the hard coded strings to `repo_arguments`.

Reply:
Updated in [86cbce5](https://github.com/roskakori/pygount/pull/111/commits/86cbce5e2f49caf80242b97c3aa5c0708b4abf03)